src/typesetting/imageTrace.md

```diff
@@ -0,0 +1,82 @@
+# Masking using AI and PS
+## Procedure
+
+I assume you have installed the [AI2ASS][]. 
+If not do it. 
+
+Drag and drop the final mask image which 
+you have created in PS into the AI. 
+Make sure you have selected **Essentials Classic**
+as work space.
+
+![Illustrator-01](images/Masking/Illustrator-01.png)
+*Work space*
+
+Now click on *Image Trace* which is present
+in bottom of the right panel. 
+After selecting you will see number of options 
+in which select **High Fidilety Photo**.[^1] 
+There is some pre-processing that needs to be done. 
+Wait for it to get completed. 
+After that you should see a window like below. 
+Now click on *Expand* in bottom of right panel. 
+
+![Illustrator-02](images/Masking/Illustrator-02.png)
+*Expand*
+
+After Expansion you should see blue lines 
+on image like below image. 
+Now click on *Ungroup* in bottom of right panel. 
+
+![Illustrator-03](images/Masking/Illustrator-03.png)
+*Ungroup*
+
+Deselect the whole thing by clicking anywhere outside 
+the canvas and press *Ctrl+Shift+d*.[^2] 
+After that click on white background and press *Delete*. 
+If you didn't messed up you should see a clean image.
+
+![Illustrator-04](images/Masking/Illustrator-04.png)
+*Clean AI image*
+
+Sometimes ImageTrace can create 
+**white spots around the edges**, 
+so it is better to do a sanity check to make sure 
+that there are no white spots around the edges. 
+Click on *Document Setup* in bottom right. 
+In the pop up window select *Grid Colors* option to *Custom*
+and chose a color which is in contrast with the image.
+I generally use green but use whatever color
+is convienent for you.
+And click ok.
+
+![Illustrator-08](images/Masking/Illustrator-08.png)
```

Comment:
Because the enumeration of the images is evidently not in order, please use more descriptive names for the files, which will also be more future-proof.

Reply:
Made the file name more clear now and used respective directories for ease